exchange_calendars/exchange_calendar_xtai.py

```diff
@@ -192,13 +216,13 @@ def bridge_fri(dt: datetime.datetime) -> datetime.datetime | None:
     "National Day of the Republic of China extra Monday",
     month=10,
     day=10,
-    observance=bridge_mon,
+    observance=partial_bridge_mon,
```

Comment:
Could you provide a reference (within the PR comments is fine) confirming that this extra Monday is not applied from 2024.